lazyflow/operators/tiktorch/classifier.py

```diff
@@ -73,6 +73,44 @@ def input_axes(self):
     def output_axes(self):
         return self.__session.outputAxes
 
+    @property
+    def offset(self):
+        return self.__session.offset
+
+    @property
+    def scale(self):
+        return self.__session.scale
+
+    @property
+    def output_shapes(self):
+        return self.__session.validShapes
+
+    @property
+    def output_shape(self):
+        """
+        shape = shape(input_tensor) * scale + 2 * offset
+        """
+        offset_by_name = {d.name: d.size for d in self.offset}
+        scale_by_name = {d.name: d.size for d in self.scale}
+
+        result = []
+        for shape in self.__session.validShapes:
```

Comment:
We don't need to loop through everything if only `result[0]` is returned (unless there's something else going on here).

Reply:
that's True. Honestly, I'm not sure how to best handle this. The model will give a list of valid shapes, and on the ilastik side, we might choose one depending on whether we do batch prediction, interactive prediction and so on. I implemented it like this in order to be able to switch quickly once we figure out what we want to do with it.

Reply:
Fair point. I think adding a FIXME or some other comment about this would be nice.